Projects/Server/World/World.cs

```diff
@@ -38,15 +38,14 @@ public enum WorldState
 
 public static class World
 {
-    private static ILogger logger = LogFactory.GetLogger(typeof(World));
+    private static readonly ILogger logger = LogFactory.GetLogger(typeof(World));
 
     private static readonly ItemPersistence _itemPersistence = new();
     private static readonly MobilePersistence _mobilePersistence = new();
     private static readonly GenericEntityPersistence<BaseGuild> _guildPersistence = new("Guilds", 3, 1, 0x7FFFFFFF);
 
     private static int _threadId;
     private static readonly SerializationThreadWorker[] _threadWorkers = new SerializationThreadWorker[Math.Max(Environment.ProcessorCount - 1, 1)];
-    private static readonly ManualResetEvent _diskWriteHandle = new(true);
```

Comment:
Let's put this back and only reset once files are flushed, and archives are run.